system/View/Exceptions/ViewException.php

```diff
@@ -15,36 +15,57 @@
 
 class ViewException extends FrameworkException
 {
+    /**
+     * @return ViewException
```

Comment:
Here and below please use the polymorphic form:
```suggestion
     * @return static
```

Reply:
Ok.